config/style_check_hook.sh

```diff
@@ -115,12 +117,12 @@ case "${1}" in
         exit 0
         ;;
     --last)
-        FILES=$(git diff --name-only remotes/origin/master...HEAD)
+        FILES=$(git diff --name-only remotes/origin/"${CHANGE_TARGET}"...HEAD)
         LAST="True"
         FIX="false"
         ;;
     --branch-fix)
-        FILES=$(git diff --name-only remotes/origin/master...HEAD)
+        FILES=$(git diff --name-only remotes/origin/"${CHANGE_TARGET}"...HEAD)
```

Comment:
all of the changes in this file are on purpose - hotfix for checking against the correct branch